package-lock.json

```diff
@@ -4,12 +4,6 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "@uirouter/core": {
```

Comment:
Please remove this file from your commit.

Reply:
what you commented on already exist on develop, not on my PR